package.json

```diff
@@ -11,7 +11,8 @@
     "postinstall": "patch-package",
     "lint": "wsrun -c lint",
     "build": "wsrun -te -c build",
-    "test": "wsrun -c test"
+    "test": "wsrun -x @ethereum-waffle/jest -c test",
```

Comment:
test
test:jest
test:others

Reply:
@marekkirejczyk Any plan to bring back jest?

Reply:
Generally we will deprecate JEST support, as it didn't get traction.
Current version supports JEST.